core/services/vrf/vrf_fulfillment_cost_test.go

```diff
@@ -22,7 +22,7 @@ func TestMeasureFulfillmenttGasCost(t *testing.T) {
 		coordinator.rootContractAddress, coordinator.coordinatorABI,
 		"fulfillRandomnessRequest", proofBlob[:])
 
-	assert.Greater(t, estimate, uint64(148000),
+	assert.Greater(t, estimate, uint64(145000),
```

Comment:
Why this change?

Reply:
Optimization lowered the best case for gas usage.